internal/user/repository/view/active_user_ids_by_session_id.sql

```diff
@@ -0,0 +1,11 @@
+SELECT
+    s.user_agent_id,
+    s.user_id
+FROM auth.user_sessions s
+    JOIN auth.user_sessions s2
```

Comment:
Why do you do a JOIN here?

Reply:
i need to either self join or sub query for the `user_agent_id` of the passed `id` to get all `user_id`s